app/views/projects/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<%-# TODO: tables should be variable width depending on how many years of data we have -%>
+
+<h1><%= @project.name %></h1>
+
+<table>
+  <tr>
+    <th>Year</th>
+    <% @project.years.each do |year| %>
+      <th><%= year.date %></th>
+    <% end %>
+  </tr>
+  <% @project.years.each do |year| %>
+    <tr>
```

Comment:
It might be cleaner to nest all the <%= yada yada %> within the do-loop so you don't need multiple loops.